src/modules/0.0.20/iasql_functions/index.ts

```diff
@@ -5,6 +5,10 @@ import { IasqlOperationType } from './entity';
 class IasqlFunctions extends ModuleBase {
   constructor() {
     super();
+    this.sql = {
+      afterInstallSqlPath: 'sql/create_fns.sql',
+      beforeUninstallSqlPath: 'sql/drop_fns.sql',
+    };
```

Comment:
Oh, I thought these were gonna be inferred?

Reply:
well I have left both options, if it is here, takes priority, if not is inferred in `sql/after_install.sql` or `sql/before_uninstall.sql`

Reply:
Ah, right, and you're exercising both paths, here and in `aws_memory_db`